app/src/main/res/values/strings.xml

```diff
@@ -66,129 +66,129 @@
-    <string name="size_title">Size</string>
+    <string name="size_title">Dot size</string>
     <string name="pick_color">Pick color</string>
     <string name="select">Select</string>
     <string name="pick_dot_color">Pick dot color</string>
-    <string name="dot_mic">Dot microphone</string>
+    <string name="dot_mic">Dot mic</string>
     <string name="dot_camera">Dot camera</string>
     <string name="dot_indicator">Dot indicator</string>
-    <string name="top_right_corner">Top right corner</string>
-    <string name="top_left_corner">Top left corner</string>
-    <string name="center_right_side">Center right side</string>
-    <string name="center_left_side">Center left side</string>
-    <string name="bottom_right_corner">Bottom right corner</string>
-    <string name="bottom_left_corner">Bottom left corner</string>
+    <string name="top_right_corner">Top-right corner</string>
+    <string name="top_left_corner">Top-left corner</string>
+    <string name="center_right_side">Center-right side</string>
+    <string name="center_left_side">Center-left side</string>
+    <string name="bottom_right_corner">Bottom-right corner</string>
+    <string name="bottom_left_corner">Bottom-left corner</string>
     <string name="top_center_horizontal">Center top</string>
     <string name="bottom_center_horizontal">Center bottom</string>
-    <string name="dot_position">Dot position on screen</string>
+    <string name="dot_position">Position on screen</string>
     <string name="cancel">Cancel</string>
     <string name="discard_changes">Discard</string>
     <string name="save">Save</string>
-    <string name="save_progress_expl">Discard or save your current changes?</string>
-    <string name="customize_cam_or_mic">Customize camera or microphone parameters?</string>
-    <string name="saved">Saved!</string>
+    <string name="save_progress_expl">Discard or save current changes?</string>
+    <string name="customize_cam_or_mic">Customize camera and/or mic parameters?</string>
+    <string name="saved">Saved</string>
     <string name="disk_status">Disk status</string>
-    <string name="crash_copied_to_clipboard">Crash details copied to clipboard!</string>
-    <string name="content_copied_to_clipboard">Copied to clipboard!</string>
-    <string name="web_browser_required">Web browser required for this action!</string>
-    <string name="informational">Informational</string>
+    <string name="crash_copied_to_clipboard">Crash details copied to clipboard</string>
+    <string name="content_copied_to_clipboard">Copied to clipboard</string>
+    <string name="web_browser_required">Install a web browser first.</string>
+    <string name="informational">Info</string>
     <string name="customizations">Customizations</string>
-    <string name="home_page">Home page</string>
-    <string name="license" translatable="false">GPL 3.0</string>
+    <string name="home_page">Homepage</string>
+    <string name="license" translatable="false">GPLv3+</string>
```

Comment:
Assumption on my part.
F-Droid metadata specifies "only" in cases of neither being specified.